p2p/src/lib.rs

```diff
@@ -37,14 +37,14 @@ pub enum Recipients<P: PublicKey> {
     One(P),
 }
 
-/// Interface for sending messages to a set of recipients.
-pub trait Sender: Debug + Clone + Send + Sync + 'static {
-    /// Error that can occur when sending a message.
-    type Error: Debug + StdError + Send + Sync;
-
+/// Interface for sending messages to a set of recipients without rate-limiting restrictions.
+pub trait UnlimitedSender: Clone + Send + Sync + 'static {
```

Comment:
I think `Unbounded` is more common in rust but understand it plays with "rate limiting" and `limitedsender`